src/app/organization/offices/create-offices/create-offices.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="container" fxFlex="50%">
```

Comment:
Use CSS to modify the container width instead of flex. (see Accounting -> Closing Entries -> Create)